apps/emqx_resource/include/emqx_resource.hrl

```diff
@@ -154,7 +154,7 @@
 %% boolean
 -define(START_AFTER_CREATED, true).
 
--define(TEST_ID_PREFIX, "_probe_:").
```

Comment:
https://github.com/emqx/hocon/blob/master/src/hocon_tconf.erl#L81
map key need to be `"^[A-Za-z0-9]+[A-Za-z0-9-_]*$"` so we change the prefix to t_probe_

Reply:
is `:` needed ?

Reply:
the places that concatenate this with the name don't add a `:`, but I assume it'll fail hocon's key format check